shotover-proxy/examples/cassandra-protect-local/topology.yaml

```diff
@@ -0,0 +1,64 @@
+---
+sources:
+  cassandra_prod:
+    Cassandra:
+      query_processing: true
+      listen_addr: "127.0.0.1:9042"
+      cassandra_ks:
+        system.local:
+          - key
+        test.simple:
+          - pk
+        test.clustering:
+          - pk
+          - clustering
+chain_config:
+  main_chain:
+    - Protect:
+        key_manager:
+          Local:
+            kek:
```

Comment:
I think yaml allows us to put the raw bytes in like a json array format which is a bit easier on the eyes `[ 30, 223, 12 ... ]`

Reply:
Personally I would just leave it as is otherwise we have a 160 character wide line.
@conorbros which would you pick?

Reply:
I saw that and thought there would be a better way to do it (perhaps formatting it as hex?) but decided to leave it to not block this PR and get the tests in.

Reply:
Alright, I kept it on one line and hexed the values.

Reply:
I meant as a long hex string.

Reply:
Not sure if it's possible to do that, but it seems strange to have to a supply a key byte by byte. I'd imagine you'd have it as a hex or base64 encoded string instead.

Reply:
oh is it possible to write an array of integers as a hex string in yaml?

Reply:
I was thinking we read it as an encoded string and then convert into the bytes format that `sodiumoxide` expects.

Reply:
ah right, I see what you mean.
That would certainly be nicer, but as you said earlier, lets address that later.

Reply:
Opened an issue for it: #510